README.md

```diff
@@ -1 +1 @@
-# my-codereview
```

Comment:
Kindly update your README.md file. It should include a project description, the author(s) of the project, contributors and acknowledgements.